### DISCUSSION TITLE
Multiple/compound condition?

### DISCUSSION THREAD

**Discussion by dels07** [statelyai/xstate#2279] [Q&A]
Hi,

I'm fairly new to xstate, I want to ask if we can use multiple cond, so I can make my guards class modular, consider this simple example
```
{
// ...
  states: {
    ready_to_ship: {
      on: {
        qc_failed: { 
          target: '#process_order.item_preparation',
          cond: ['isItemQced', 'isQcFailed']
        },
        qc_passed: {
          target: '#ship_to_customer.item_shipment',
          cond: ['isItemQced', 'isQcSuccess', 'isPacked']
        }
  } 
}
```
so the guard condition can be modular/reusable without need to make specific guard for each condition

Regards,
Dels

**Comment by davidkpiano** [comment]
Unfortunately not in v4; this will be available in v5 though. For now, you have to make custom guards that encompass all of those guards.

I'll let you know if we introduce any features that allow you to accomplish this in v4, though.
